boards/FSM/Inc/Io/Io_MSP3002K5P3N1.h

```diff
@@ -1,15 +1,27 @@
 #pragma once
 
+#include <stdbool.h>
+
 /**
  * Check if the pressure from MSP-300-2K5-P-3-N-1 exceeds the brake pressure
  * threshold. If the pressure exceeds threshold, the brake is said to be
- * actuated.
+ * actuated
  * @return True if the brake is actuated, false if the brake is not actuated
  */
 bool Io_MSP3002K5P3N1_IsBrakeActuated(void);
 
 /**
- * Get the pressure in psi from the MSP-300-2K5-P-3-N-1 brake pressure sensor
+ * Check if the MSP-300-2K5-P-3-N-1 is in an open circuit or short circuit
+ * state. If the output voltage exceeds the upper threshold or goes below the
+ * minimum threshold, the brake pressure sensor is said to be open/short
+ * circuited
```

Comment:
I don't think this is necessary. Previously it wasn't clear what brake is "actuated" means. But here open-circuit and short-circuit are well-defined and unambiguous.
```suggestion
 * state.
```